### PR TITLE
Update Fifth Third

### PR DESCRIPTION
**Pull request by rdgarfinkel**
Add Fifth Third. Just had to log in with a code sent to my cell phone, hadn't logged in on this device.

### CODE REVIEW

File: _data/banking.yml [hunk 1]
```diff
@@ -522,6 +522,8 @@ websites:
     twitter: FifthThird
     facebook: FifthThirdBank
```
Comment (phallobst) [inline _data/banking.yml]:
```suggestion
```

**Review by phallobst — CHANGES_REQUESTED** [review]
Is there a public documentation page for 2FA available?

**Comment by rdgarfinkel** [comment]
> Is there a public documentation page for 2FA available?

I don’t see one in looking for about five minutes

**Comment by rdgarfinkel** [comment]
I got "fatal: reference is not a tree: (code)
##[error]Git checkout failed with exit code: 128" in Jekyll when the test ran. I can't find where to run it again.

**Review by phallobst — APPROVED** [review]

**Comment by jamcat22** [comment]
@rdgarfinkel, since it sounds like no public-facing documentation is available, could you please provide screenshots of the 2FA setup process and/or login process so we can verify the entry?

**Please redact/crop out any personal information from your screenshots.**

You can drop or paste screenshots directly into a GitHub comment.

Thanks!

**Comment by rdgarfinkel** [comment]
Upon login to on a new computer (incognito mode):
1: image
2:
image
3:
image
4:
image
5:
image

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**